src/XMakeBuildEngine/UnitTests/FileLogger_Tests.cs

```diff
@@ -375,7 +381,7 @@ private void VerifyFileContent(string file, string expectedContent)
             }
         }
 
-        #region DistributedLogger
+#region DistributedLogger
```

Comment:
I don't know what insists on making these whitespace changes but can you revert them?  (it's been doing it to me too . . .)